modules/apps/portal-crypto-hash/portal-crypto-hash-test/src/testIntegration/java/com/liferay/portal/crypto/hash/test/CryptoHashTest.java

```diff
@@ -326,44 +324,134 @@ public void testCryptoHashVerifierWithStaticInput() throws Exception {
 						"JDJhJDEwJHVxZVh5YjF1dUdHZjZ2UWtvalljU08="))));
 	}
 
-	protected <S, R, E extends Throwable> R callService(
-			BundleContext bundleContext, Class<S> serviceClass,
-			String filterString, UnsafeFunction<S, R, E> unsafeFunction)
-		throws E {
+	private Configuration _addCryptoFactoryConfiguration(
```

Comment:
Ahh I see. In the original test, Carlos had this method in base class, but named as createFactoryConfiguration:
https://github.com/liferay-appsec/liferay-portal/commit/6405aab7247c8ee90e9f8610c1f073510017725f#diff-6b57f70e9685d09699715780a7c1baaa34039e72a769d8a70e913f123dc7f775R203-R213
That's why we have two same name methods after merging base test to current test.